app/views/layouts/_header.html.erb

```diff
@@ -64,9 +64,9 @@
         </li>
       </ul>
 
-      <form id="searchform" class="navbar-form navbar-left" role="search" autocomplete="off">
+      <form action=<%= search_query_path %> method="get" id="searchform" class="navbar-form navbar-left" role="search" autocomplete="off">
```

Comment:
ah - i think this needs quotation marks around the `action` parameter -- almost there though!